packages/product-components/src/components/TokenIconSet/TokenIconSet.tsx

```diff
@@ -49,9 +52,9 @@ export const TokenIconSet = ({ network, tokens }: TokenIconSetProps) => {
     return (
         <IconContainer $length={length}>
             {length > 3 && <TokenIconPlaceholder $elevation={elevation} />}
-            {visibleTokens.map(token => (
+            {visibleTokens.map((token, i) => (
```

Comment:
not needed as duplicated cannot be there right?

Reply:
Yes, not anymore https://github.com/trezor/trezor-suite/pull/14996/commits/78aaf22d419fdf3f33b44af2ded70d95baf30f2d